test/remote/gateways/remote_stripe_apple_pay_test.rb

```diff
@@ -103,7 +103,9 @@ def test_purchase_with_unsuccessful_apple_pay_token_exchange
   def test_successful_purchase_with_apple_pay_raw_cryptogram
     credit_card = network_tokenization_credit_card('4242424242424242',
       payment_cryptogram: "EHuWW9PiBkWvqE5juRwDzAUFBAk=",
-      verification_value: nil
+      verification_value: nil,
+      eci: '05',
```

Comment:
Will this test now fail if no `eci` or `source` is provided? (thinking about backwards compatibility w/ existing clients and, if we want to be backwards compatible, we should probably keep a test w/o these values)?